src/jackdaw/test/transports/kafka.clj

```diff
@@ -0,0 +1,201 @@
+(ns jackdaw.test.transports.kafka
+  (:require
+   [clojure.core.async :as async]
+   [clojure.tools.logging :as log]
+   [jackdaw.client :as kafka]
+   [jackdaw.data :as jd]
+   [jackdaw.test.commands :as cmd]
+   [jackdaw.test.transports :as t]
+   [jackdaw.test.serde :refer [apply-serializers apply-deserializers
+                               serde-map
+                               byte-array-serde]])
+  (:import
+   org.apache.kafka.streams.KafkaStreams$StateListener
+   org.apache.kafka.clients.consumer.ConsumerRecord
+   org.apache.kafka.clients.producer.ProducerRecord))
+
+(defn subscribe
+  "Subscribes to specified topics
+
+   `consumer` should be a kafka consumer
+   `topic-config` should be a sequence of topic-metadata maps"
+  [consumer topic-config]
+  (kafka/subscribe consumer topic-config))
+
+(defn load-assignments
+  [consumer]
+  (.poll consumer 0)
+  (.assignment consumer))
+
+(defn seek-to-end
+  "Seeks to the end of all the partitions assigned to the given consumer
+   and returns the updated consumer"
+  [consumer & topic-partitions]
+  (let [assigned-partitions (or topic-partitions (load-assignments consumer))]
+    (.seekToEnd consumer assigned-partitions)
+    (doseq [assigned-partition assigned-partitions]
+      ;; This forces the seek to happen now
+      (.position consumer assigned-partition))
+    consumer))
```

Comment:
This looks similar to `jackdaw.client/seek-to-end-eager`. Can we use that instead?

https://github.com/FundingCircle/jackdaw/blob/master/src/jackdaw/client.clj#L218